.travis.yml

```diff
@@ -24,6 +24,14 @@ jobs:
       script:
         - (cd proxy && cargo test --locked --no-default-features)
 
+    - language: rust
```

Comment:
TIOLI: should we maybe name this build job "benchmark" or something, so that it's  clearer it's not a regular build?

Reply:
i don't think that's possible without making it its own stage, which i don't want to do.

Reply:
ah, nevermind, I thought it was its own stage.